### PR TITLE
add korg deployment

### PR DESCRIPTION
**Pull request by code-sleuth**

### CODE REVIEW

File: sources/sds-deployment.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ environments:
...
+      SDS_TASKS_SKIP_RE: '^sds-(lfn-odl-gerrit|open-mainframe-project|accord|acr|acrn|allseen|alljoyn|act|agl|bitcoin-protocol|cc|CC|cephfoundation|chaoss|ojsf-chassis|chips|cip|clusterduck|cni|caf|ccc|cdf|cord-datacenter|cord-central-office-re-architected-as-a-datacenter|cregit|datapractices|dent|desktopwg|easycla|elisa|finops|finos|fossbazaar|foundationdb|frr|gql|internet-security-research-group|interuss|iovisor|janusgraph|kernelci|openkinetic|laminas|lets-encrypt|lf-asia-llc|lfph|lsb|linuxboot|llvmlinux|ltsi|lvfs|mapzen|mojaglobal|newticity|onf|O3DE|o3de|a11y|open-alliance-for-cloud-adoption|opencontainers|open-network-operating-system-onos|osecc|osapa|ova|open-voice-network|ovs|openapi|openbel|openbmc|openchain|OEEW|oeew|openfabrics-inc|openhpc|ojsf|openmama|openmessaging|openpowerfoundation|opensds|osquery|patentcommons|quartermaster|rcons|real-time-linux|redteam|rtdb|riscv|risc-v-international|sel4|soda-foundation|sof|tlf|openssf|reactive|tizen|ucf)-'   # specifying which tasks should be skipped, can be something like '^sds-(foundation1|foundation2|...|foundationN)-'
```
Comment (1010sachin) [inline sources/sds-deployment.yaml:19]: why did we remove ```cncf``` from the ```SDS_TASKS_SKIP_RE```? Also we will ```korg``` added in here too right?

File: sources/sds-deployment.yaml [hunk 2]
```diff
@@ -44,7 +58,7 @@ environments:
...
+      SDS_TASKS_RE: '^sds-(lfn-odl-gerrit|open-mainframe-project|accord|acr|acrn|allseen|alljoyn|act|agl|bitcoin-protocol|cc|CC|cephfoundation|chaoss|ojsf-chassis|chips|cip|clusterduck|cni|caf|ccc|cdf|cord-datacenter|cord-central-office-re-architected-as-a-datacenter|cregit|datapractices|dent|desktopwg|easycla|elisa|finops|finos|fossbazaar|foundationdb|frr|gql|internet-security-research-group|interuss|iovisor|janusgraph|kernelci|openkinetic|laminas|lets-encrypt|lf-asia-llc|lfph|lsb|linuxboot|llvmlinux|ltsi|lvfs|mapzen|mojaglobal|newticity|onf|O3DE|o3de|a11y|open-alliance-for-cloud-adoption|opencontainers|open-network-operating-system-onos|osecc|osapa|ova|open-voice-network|ovs|openapi|openbel|openbmc|openchain|OEEW|oeew|openfabrics-inc|openhpc|ojsf|openmama|openmessaging|openpowerfoundation|opensds|osquery|patentcommons|quartermaster|rcons|real-time-linux|redteam|rtdb|riscv|risc-v-international|sel4|soda-foundation|sof|tlf|openssf|reactive|tizen|ucf)-'
```
Comment (1010sachin) [inline sources/sds-deployment.yaml:61]: why did we remove cncf from the SDS_TASKS_RE? Also we will korg added in here too right?

Comment (code-sleuth) [reply sources/sds-deployment.yaml]: done

File: sources/sds-deployment.yaml (hunk 1 above)
Comment (code-sleuth) [reply sources/sds-deployment.yaml]: done

File: sources/sds-deployment.yaml [hunk 3]
```diff
@@ -31,6 +31,20 @@ environments:
       SDS_SCROLL_SIZE: 1000
       SDS_ES_BULKSIZE: 1000
       SDS_TASKS_RE: '^sds-cncf-'        # the opposite of the main task - we specify which tasks should run in this deployment (instead of specifying which tasks should be skipped as in prod-main deployment)
+  - name: prod-korg
+    cron: '0 8 * * *'                   # attempt sync at 8 AM UTC, nothing will happen if the previous sync still runs
```
Comment (lukaszgryglicki) [inline sources/sds-deployment.yaml:35]: I'll change 8:00 AM to something other post-merge, I just want it to start at a different time than CNCF.

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm

**Pull request merged**